frontend/src/page/Realm.tsx

```diff
@@ -4,15 +4,15 @@ import { Link } from "react-router-dom";
 import { Breadcrumbs } from "../ui/Breadcrumbs";
 
 
-type Props = {
-    path: string[],
-};
+interface Props {
+    path: string[];
+}
```

Comment:
When researching ways to specify React props with TS, I found that `type` is recommended over `interace` for these purposes. Right now, I can't find the article where I first read it but the search term "react typescript props type vs interface" brings up a couple of articles that state the same. 

> Hopefully, this article [.. ] convinced you that type aliases are preferable for React applications. [...] go ahead and disable interface-over-type-literal setting in TS Lint and start using type aliases in your React applications!

Reply:
Here is another resource: https://github.com/typescript-cheatsheets/react#types-or-interfaces

Reply:
I mean I can't say anything about the first article, since you only quoted the conclusion. The second one refers to TS 2.7, which is 2 major versions old now. Researching a bit myself reveals that there is a lot of misunderstanding and misinformation about this. The non-up-to-date TS docs don't help either.

The situation seems to be this: The differences between the two are mostly either nonexistent anymore, or don't apply in this case anyway, except for one, which is also the only argument I found somewhat convincing: `interface`-declarations with the same name are potentially sometimes maybe merged, while `type`-definitions just error in that case.

It convinced me enough, though, to question that ESLint rule entirely. Like, that's a very specific thing you **might** want to opt into **sometimes**, but it certainly shouldn't be the default. o.O

So yes, even though I call into question the specific points you bring up, I actually agree with disabling the rule. xP

Reply:
Okay :D sorry for not linking the first article, totally forgot. Thanks for your summary.